### ISSUE TITLE
Nuxt 3 Support

### ISSUE THREAD

**Issue by harlan-zw** [nuxt-community/proxy-module#118]
Hey :wave: It looks like this module hasn't been updated for Nuxt 3. In an effort to improve the developer experience of modules, I've updated the module to clarify that it only supports Nuxt 2.

If Nuxt 3 support is added it will be moved to the https://github.com/nuxt-modules organisation.

Alternatives:

- Nitro proxy using Route Rules

```ts
export default defineNuxtConfig({
  routeRules: {
    '/proxy/example': { proxy: 'https://example.com' },
    '/proxy/**': { proxy: '/api/**' },
  }
})
```

@ricardogobbosouza With Nitro supporting proxying natively, do you think it's worth deprecating this module?

Please let us know and thanks for your work!

**Comment by ricardogobbosouza** [comment]
@harlan-zw I believe that deprecating this module is the best option.
Whenever possible, use native nuxt3 resources.
What would @danielroe recommend?

**Comment by harlan-zw** [comment]
Worth noting we don't need to deprecate it, instead we can just state that it won't be upgraded to Nuxt 3*

**Comment by pi0** [comment]
Yes, it should be deprecated fully. Nitro provides both dev time and build time proxy capabilities which completely replaces functionality of this module.

If we want to consider longer term Nuxt 2 maintenance, might as well move to nitro-bridge or migrate to unjs/httpxy which is at least actively maintained.
